packages/app/src/components/common/entities/WebViewWithInjected/hooks/useWebViewHistory.ts

```
@@ -0,0 +1,48 @@
+    return () => {
+      backHandler.remove();
+    };
+  }, [webViewRef, canGoBack]);
+
+  const onNavigationStateChange = (
+    navState: WebViewNavigation,
+    onNavigate?: (arg: WebViewNavigation) => void,
+  ) => {
+    setCanGoBack(navState.canGoBack);
+    setCanGoForward(navState.canGoForward);
+    onNavigate && onNavigate(navState);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)